ouroboros-consensus-cardano/src/Ouroboros/Consensus/Cardano/CanHardFork.hs

```diff
@@ -331,6 +342,66 @@ data ShelleyPartialLedgerConfig era = ShelleyPartialLedgerConfig {
     }
   deriving (Generic, NoThunks)
 
+instance Era era => Serialise (ShelleyPartialLedgerConfig era) where
+  decode =
+    ShelleyPartialLedgerConfig
+      <$> (ShelleyLedgerConfig
+              <$> decode
+              <*> (SL.Globals
+                    dummyEpochInfo
```

Comment:
perhaps I should add a comment referring to the relevant comment in `encode` below.